cas/ctg_store.c

```diff
@@ -1274,7 +1274,7 @@ static int ctg_op_exec(struct m0_ctg_op *ctg_op, int next_phase)
 		m0_be_op_done(beop);
 		break;
 	}
-	ctg_op->co_rc = rc;
+	ctg_op->co_rc = M0_RC(rc);
```

Comment:
Do we really need this Log?? This function might be getting called manier times, I think we can skip this log.

Reply:
@shashank-parulekar  can you confirm this comment??

Reply:
Yes this log is needed only if (rc != 0) and then also you should use M0_ERR() and not M0_RC().